tempus_fugit_minecraft/model.py

```diff
@@ -55,14 +55,14 @@ class Model(object):
 
     def __init__(self) -> None:
         TEXTURE_PATH = 'assets/texture.png'
-        
+
         # A Batch is a collection of vertex lists for batched rendering.
         self.batch = Batch()
 
         # A TextureGroup manages an OpenGL texture.
         self.group = TextureGroup(image.load(TEXTURE_PATH).get_texture())
 
-        # A mapping from position to the block at that position.
+        # A mapping from position to the texture of the block at that position.
```

Comment:
Please undo this change, as of 5894f9d, the `self.world` property is a map of position tuples to Block objects, not to _just_ their textures.

Reply:
Will do.